src/authors/authors.scss

```diff
@@ -0,0 +1,103 @@
+*{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+font-family: $lato-font, sans-serif;
+  color:$white-color;
+}
+
+
+body{
```

Comment:
You have styling for the body in `styles.scss` already. Remove it. It is redundant.